src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -468,19 +464,19 @@ def open_archive_dialog(self, slack_request: SlackActionRequest, group: Group) -
 
         if slack_request.data.get("channel"):
             callback_id["channel_id"] = slack_request.data["channel"]["id"]
-        callback_id = json.dumps_experimental("integrations.slack.enable-orjson", callback_id)
+        callback_id = orjson.dumps(callback_id).decode()
 
         slack_client = SlackClient(integration_id=slack_request.integration.id)
         modal_payload = self.build_archive_modal_payload(callback_id)
         try:
             payload = {
-                "view": json.dumps_experimental("integrations.slack.enable-orjson", modal_payload),
+                "view": orjson.dumps(modal_payload).decode(),
                 "trigger_id": slack_request.data["trigger_id"],
             }
             headers = {"content-type": "application/json; charset=utf-8"}
             slack_client.post(
                 "/views.open",
-                data=json.dumps_experimental("integrations.slack.enable-orjson", payload),
+                data=orjson.dumps(payload).decode(),
```

Comment:
Afaict this is better:

```suggestion
                data=orjson.dumps(payload),
```